net-misc/netpipe/netpipe-1.0.0_beta2-r2.ebuild

```diff
@@ -8,13 +8,11 @@ inherit toolchain-funcs
 DESCRIPTION="tool to reliably distribute binary data using UDP broadcasting techniques"
 HOMEPAGE="https://www.wudika.de/~jan/netpipe/"
 SRC_URI="https://www.wudika.de/~jan/${PN}/${PN}.tar.gz -> ${P}.tar.gz"
```

Comment:
The homepage and the SRC_URI seems to be completely gone, can you check if they moved somewhere else?